src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    protected String value;
+    protected boolean isPrivate;
```

Comment:
`protected` is suitable here. We want the class, the package, and the subclasses, except the world, to be able to access this variable